original/fullkey.en.json

```diff
@@ -40,6 +41,7 @@
   "all languages": "all languages",
   "allow": "allow",
   "Always dreamed of a collaborative library for your coworking / association / book club? Here it is!": "Always dreamed of a collaborative library for your coworking / association / book club? Here it is!",
+  "download your profile description": "an export of all information stored about you, inside a JSON file. For more exports, see the \"Data\" section of the current settings",
```

Comment:
in the original `fullkey` file, the key and value are expected to be the same: you can either replace that key with the value, or move it to shortkey as `download_your_profile_description` (underscores required)